getting_started/6_Understanding_Qubits_and_Quantum_Gates.ipynb

```diff
@@ -0,0 +1,647 @@
+    " $$ |00⟩ → |00⟩$$\n",
+    " $$ |01⟩ → |01⟩$$\n",
+    " $$ |10⟩ → |11⟩$$\n",
+    " $$ |11⟩ → |10⟩$$\n",
+    " \n",
+    " \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Lets make use of the CNOT gate in a circuit. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "T  : |0|1|\n",
+      "          \n",
+      "q0 : -I-C-\n",
+      "        | \n",
+      "q1 : -I-X-\n",
+      "\n",
+      "T  : |0|1|\n",
+      "Counter({'00': 100})\n"
+     ]
+    }
+   ],
+   "source": [
+    "qc = Circuit()\n",
+    "qc.i(0)\n",
+    "qc.i(1)\n",
+    "qc.cnot(control=0,target=1)\n",
+    "print(qc)\n",
+    "\n",
+    "\n",
+    "device = LocalSimulator()\n",
+    "task = device.run(qc, shots=100)\n",
+    "result = task.result()\n",
+    "print(result.measurement_counts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Entanglement"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Entanglement is a uniquely quantum mechanical resource that plays a key role in many of the most interesting applications of quantum computation and quantum information. \n",
+    "\n",
+    "Lets see what happens when we combine `hadamard` gate with `CNOT` gate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "T  : |0|1|\n",
+      "          \n",
+      "q0 : -H-C-\n",
+      "        | \n",
+      "q1 : ---X-\n",
+      "\n",
+      "T  : |0|1|\n",
+      "Counter({'11': 60, '00': 40})\n"
+     ]
+    }
+   ],
+   "source": [
+    "qc = Circuit()\n",
+    "qc.h(0)\n",
+    "qc.cnot(control=0,target=1)\n",
+    "print(qc)\n",
+    "\n",
+    "\n",
+    "device = LocalSimulator()\n",
+    "task = device.run(qc, shots=100)\n",
+    "result = task.result()\n",
+    "print(result.measurement_counts)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The Hadamard gate puts the top qubit in a superposition, this then acts as a control input to the `CNOT`, and the target gets inverted only when the control is $|1\\rangle$\n",
+    "\n",
+    "The output state is : $$\\frac{|00\\rangle +|11\\rangle}{\\sqrt 2}$$\n",
+    "\n",
+    "\n",
+    "This is an **entangled** state. We can see that this state has 50% probability of being $|00⟩$ and 50% chance of being $|11⟩$ and 0% chance of being in the states $|01⟩$  or  $|10⟩$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Even though both qubits are in superposition, measuring one of the qubit collapses the other qubit too - even if they far apart. \n",
```

Comment:
I'm not sure why it matters that both qubits are in superposition? I'd just say
"Measuring one of the qubits collapses the state of the other qubit, even if they are far apart.".